components/nimbus/ios/Nimbus/NimbusBuilder.swift

```diff
@@ -231,8 +231,7 @@ public class NimbusBuilder {
     // swiftlint:enable function_body_length
 
     func getCoenrollingFeatureIds() -> [String] {
-        // This will be changed to use the feature manifest in EXP-3265
-        []
+        featureManifest != nil ? featureManifest!.getCoenrollingFeatureIds() : []
```

Comment:
Nit: `??` is the same as the elvis operator.

```suggestion
        featureManifest?.getCoenrollingFeatureIds() ?? []
```

Reply:
Ahhh perfect!